[{"cve_id": "CVE-2016-2513", "cwe_id": "CWE-200", "repo_url": "https://github.com/django/django", "hash": "67b46ba7016da2d259c1ecc7d666d11f5e1cfaab", "repo_name": "django/django", "repo_language": "Python"}, {"cve_id": "CVE-2016-2533", "cwe_id": "CWE-119", "repo_url": "https://github.com/python-pillow/Pillow", "hash": "ae453aa18b66af54e7ff716f4ccb33adca60afd4", "repo_name": "python-pillow/Pillow", "repo_language": "Python"}, {"cve_id": "CVE-2016-4009", "cwe_id": "CWE-119", "repo_url": "https://github.com/python-pillow/Pillow", "hash": "4e0d9b0b9740d258ade40cce248c93777362ac1e", "repo_name": "python-pillow/Pillow", "repo_language": "Python"}, {"cve_id": "CVE-2016-4444", "cwe_id": "CWE-77", "repo_url": "https://github.com/fedora-selinux/setroubleshoot", "hash": "5cd60033ea7f5bdf8c19c27b23ea2d773d9b09f5", "repo_name": "fedora-selinux/setroubleshoot", "repo_language": "Python"}, {"cve_id": "CVE-2016-4445", "cwe_id": "CWE-77", "repo_url": "https://github.com/fedora-selinux/setroubleshoot", "hash": "2d12677629ca319310f6263688bb1b7f676c01b7", "repo_name": "fedora-selinux/setroubleshoot", "repo_language": "Python"}, {"cve_id": "CVE-2016-5851", "cwe_id": "CWE-611", "repo_url": "https://github.com/python-openxml/python-docx", "hash": "61b40b161b64173ab8e362aec1fd197948431beb", "repo_name": "python-openxml/python-docx", "repo_language": "Python"}, {"cve_id": "CVE-2016-6298", "cwe_id": "CWE-200", "repo_url": "https://github.com/latchset/jwcrypto", "hash": "eb5be5bd94c8cae1d7f3ba9801377084d8e5a7ba", "repo_name": "latchset/jwcrypto", "repo_language": "Python"}, {"cve_id": "CVE-2016-6639", "cwe_id": "CWE-254", "repo_url": "https://github.com/cloudfoundry/php-buildpack", "hash": "e2db3ccd4812e0c0aba20720fc51789d981aba67", "repo_name": "cloudfoundry/php-buildpack", "repo_language": "Python"}, {"cve_id": "CVE-2016-6902", "cwe_id": "CWE-264", "repo_url": "https://github.com/ghantoos/lshell", "hash": "a686f71732a3d0f16df52ef46ab8a49ee0083c68", "repo_name": "ghantoos/lshell", "repo_language": "Python"}, {"cve_id": "CVE-2016-6903", "cwe_id": "CWE-264", "repo_url": "https://github.com/ghantoos/lshell", "hash": "e72dfcd1f258193f9aaea3591ecbdaed207661a0", "repo_name": "ghantoos/lshell", "repo_language": "Python"}, {"cve_id": "CVE-2016-9243", "cwe_id": "CWE-20", "repo_url": "https://github.com/pyca/cryptography", "hash": "b924696b2e8731f39696584d12cceeb3aeb2d874", "repo_name": "pyca/cryptography", "repo_language": "Python"}, {"cve_id": "CVE-2016-9909", "cwe_id": "CWE-79", "repo_url": "https://github.com/html5lib/html5lib-python", "hash": "9b8d8eb5afbc066b7fac9390f5ec75e5e8a7cab7", "repo_name": "html5lib/html5lib-python", "repo_language": "Python"}, {"cve_id": "CVE-2016-9910", "cwe_id": "CWE-79", "repo_url": "https://github.com/html5lib/html5lib-python", "hash": "9b8d8eb5afbc066b7fac9390f5ec75e5e8a7cab7", "repo_name": "html5lib/html5lib-python", "repo_language": "Python"}, {"cve_id": "CVE-2016-9964", "cwe_id": "CWE-93", "repo_url": "https://github.com/bottlepy/bottle", "hash": "6d7e13da0f998820800ecb3fe9ccee4189aefb54", "repo_name": "bottlepy/bottle", "repo_language": "Python"}, {"cve_id": "CVE-2017-0881", "cwe_id": "CWE-863", "repo_url": "https://github.com/zulip/zulip", "hash": "7ecda1ac8e26d8fb3725e954b2dc4723dda2255f", "repo_name": "zulip/zulip", "repo_language": "Python"}, {"cve_id": "CVE-2017-0906", "cwe_id": "CWE-918", "repo_url": "https://github.com/recurly/recurly-client-python", "hash": "049c74699ce93cf126feff06d632ea63fba36742", "repo_name": "recurly/recurly-client-python", "repo_language": "Python"}, {"cve_id": "CVE-2017-1002150", "cwe_id": "CWE-601", "repo_url": "https://github.com/fedora-infra/python-fedora", "hash": "b27f38a67573f4c989710c9bfb726dd4c1eeb929", "repo_name": "fedora-infra/python-fedora", "repo_language": "Python"}, {"cve_id": "CVE-2017-14695", "cwe_id": "CWE-22", "repo_url": "https://github.com/saltstack/salt", "hash": "80d90307b07b3703428ecbb7c8bb468e28a9ae6d", "repo_name": "saltstack/salt", "repo_language": "Python"}, {"cve_id": "CVE-2017-14696", "cwe_id": "CWE-20", "repo_url": "https://github.com/saltstack/salt", "hash": "5f8b5e1a0f23fe0f2be5b3c3e04199b57a53db5b", "repo_name": "saltstack/salt", "repo_language": "Python"}, {"cve_id": "CVE-2017-15111", "cwe_id": "CWE-59", "repo_url": "https://github.com/jdennis/keycloak-httpd-client-install", "hash": "07f26e213196936fb328ea0c1d5a66a09d8b5440", "repo_name": "jdennis/keycloak-httpd-client-install", "repo_language": "Python"}, {"cve_id": "CVE-2017-15112", "cwe_id": "CWE-200", "repo_url": "https://github.com/jdennis/keycloak-httpd-client-install", "hash": "c3121b271abaaa1a76de2b9ae89dacde0105cd75", "repo_name": "jdennis/keycloak-httpd-client-install", "repo_language": "Python"}, {"cve_id": "CVE-2017-16615", "cwe_id": "NVD-CWE-noinfo", "repo_url": "https://github.com/thanethomson/MLAlchemy", "hash": "bc795757febdcce430d89f9d08f75c32d6989d3c", "repo_name": "thanethomson/MLAlchemy", "repo_language": "Python"}, {"cve_id": "CVE-2017-16618", "cwe_id": "NVD-CWE-noinfo", "repo_url": "https://github.com/tadashi-aikawa/owlmixin", "hash": "5d0575303f6df869a515ced4285f24ba721e0d4e", "repo_name": "tadashi-aikawa/owlmixin", "repo_language": "Python"}, {"cve_id": "CVE-2017-16667", "cwe_id": "CWE-78", "repo_url": "https://github.com/bit-team/backintime", "hash": "cef81d0da93ff601252607df3db1a48f7f6f01b3", "repo_name": "bit-team/backintime", "repo_language": "Python"}, {"cve_id": "CVE-2017-16763", "cwe_id": "NVD-CWE-noinfo", "repo_url": "https://github.com/bbengfort/confire", "hash": "8cc86a5ec2327e070f1d576d61bbaadf861597ea", "repo_name": "bbengfort/confire", "repo_language": "Python"}, {"cve_id": "CVE-2017-16876", "cwe_id": "CWE-79", "repo_url": "https://github.com/lepture/mistune", "hash": "5f06d724bc05580e7f203db2d4a4905fc1127f98", "repo_name": "lepture/mistune", "repo_language": "Python"}, {"cve_id": "CVE-2017-17713", "cwe_id": "CWE-89", "repo_url": "https://github.com/boxug/trape", "hash": "628149159ba25adbfc29a3ae1d4b10c7eb936dd3", "repo_name": "jofpin/trape", "repo_language": "Python"}, {"cve_id": "CVE-2017-17714", "cwe_id": "CWE-79", "repo_url": "https://github.com/boxug/trape", "hash": "628149159ba25adbfc29a3ae1d4b10c7eb936dd3", "repo_name": "jofpin/trape", "repo_language": "Python"}, {"cve_id": "CVE-2017-2661", "cwe_id": "CWE-79", "repo_url": "https://github.com/ClusterLabs/pcs", "hash": "1874a769b5720ae5430f10c6cedd234430bc703f", "repo_name": "ClusterLabs/pcs", "repo_language": "Python"}, {"cve_id": "CVE-2017-2809", "cwe_id": "CWE-94", "repo_url": "https://github.com/tomoh1r/ansible-vault", "hash": "3f8f659ef443ab870bb19f95d43543470168ae04", "repo_name": "tomoh1r/ansible-vault", "repo_language": "Python"}, {"cve_id": "CVE-2017-5537", "cwe_id": "CWE-200", "repo_url": "https://github.com/WeblateOrg/weblate", "hash": "abe0d2a29a1d8e896bfe829c8461bf8b391f1079", "repo_name": "WeblateOrg/weblate", "repo_language": "Python"}, {"cve_id": "CVE-2017-5934", "cwe_id": "CWE-79", "repo_url": "https://github.com/moinwiki/moin-1.9", "hash": "70955a8eae091cc88fd9a6e510177e70289ec024", "repo_name": "moinwiki/moin-1.9", "repo_language": "Python"}, {"cve_id": "CVE-2017-7266", "cwe_id": "CWE-601", "repo_url": "https://github.com/Netflix/security_monkey", "hash": "3b4da13efabb05970c80f464a50d3c1c12262466", "repo_name": "Netflix/security_monkey", "repo_language": "Python"}, {"cve_id": "CVE-2017-7481", "cwe_id": "CWE-20", "repo_url": "https://github.com/ansible/ansible", "hash": "ed56f51f185a1ffd7ea57130d260098686fcc7c2", "repo_name": "ansible/ansible", "repo_language": "Python"}, {"cve_id": "CVE-2017-7572", "cwe_id": "CWE-362", "repo_url": "https://github.com/bit-team/backintime", "hash": "7f208dc547f569b689c888103e3b593a48cd1869", "repo_name": "bit-team/backintime", "repo_language": "Python"}, {"cve_id": "CVE-2017-8342", "cwe_id": "CWE-362", "repo_url": "https://github.com/Kozea/Radicale", "hash": "059ba8dec1f22ccbeab837e288b3833a099cee2d", "repo_name": "Kozea/Radicale", "repo_language": "Python"}, {"cve_id": "CVE-2017-8342", "cwe_id": "CWE-362", "repo_url": "https://github.com/Kozea/Radicale", "hash": "190b1dd795f0c552a4992445a231da760211183b", "repo_name": "Kozea/Radicale", "repo_language": "Python"}, {"cve_id": "CVE-2018-1000070", "cwe_id": "CWE-94", "repo_url": "https://github.com/Bitmessage/PyBitmessage", "hash": "3a8016d31f517775d226aa8b902480f4a3a148a9", "repo_name": "Bitmessage/PyBitmessage", "repo_language": "Python"}, {"cve_id": "CVE-2018-1000089", "cwe_id": "CWE-532", "repo_url": "https://github.com/anymail/django-anymail", "hash": "1a6086f2b58478d71f89bf27eb034ed81aefe5ef", "repo_name": "anymail/django-anymail", "repo_language": "Python"}, {"cve_id": "CVE-2018-1000559", "cwe_id": "CWE-79", "repo_url": "https://github.com/qutebrowser/qutebrowser", "hash": "5a7869f2feaa346853d2a85413d6527c87ef0d9f", "repo_name": "qutebrowser/qutebrowser", "repo_language": "Python"}, {"cve_id": "CVE-2018-1000807", "cwe_id": "CWE-416", "repo_url": "https://github.com/pyca/pyopenssl", "hash": "e73818600065821d588af475b024f4eb518c3509", "repo_name": "pyca/pyopenssl", "repo_language": "Python"}, {"cve_id": "CVE-2018-1000809", "cwe_id": "CWE-20", "repo_url": "https://github.com/privacyidea/privacyidea", "hash": "a3edc09beffa2104f357fe24971ea3211ce40751", "repo_name": "privacyidea/privacyidea", "repo_language": "Python"}, {"cve_id": "CVE-2018-10657", "cwe_id": "CWE-20", "repo_url": "https://github.com/matrix-org/synapse", "hash": "33f469ba19586bbafa0cf2c7d7c35463bdab87eb", "repo_name": "matrix-org/synapse", "repo_language": "Python"}, {"cve_id": "CVE-2018-10895", "cwe_id": "CWE-352", "repo_url": "https://github.com/qutebrowser/qutebrowser", "hash": "43e58ac865ff862c2008c510fc5f7627e10b4660", "repo_name": "qutebrowser/qutebrowser", "repo_language": "Python"}, {"cve_id": "CVE-2018-10897", "cwe_id": "CWE-59", "repo_url": "https://github.com/rpm-software-management/yum-utils", "hash": "6a8de061f8fdc885e74ebe8c94625bf53643b71c", "repo_name": "rpm-software-management/yum-utils", "repo_language": "Python"}, {"cve_id": "CVE-2018-10897", "cwe_id": "CWE-59", "repo_url": "https://github.com/rpm-software-management/yum-utils", "hash": "7554c0133eb830a71dc01846037cc047d0acbc2c", "repo_name": "rpm-software-management/yum-utils", "repo_language": "Python"}, {"cve_id": "CVE-2018-11757", "cwe_id": "NVD-CWE-noinfo", "repo_url": "https://github.com/apache/incubator-openwhisk-runtime-docker", "hash": "891896f25c39bc336ef6dda53f80f466ac4ca3c8", "repo_name": "apache/openwhisk-runtime-docker", "repo_language": "Python"}, {"cve_id": "CVE-2018-14650", "cwe_id": "CWE-732", "repo_url": "https://github.com/sosreport/sos-collector", "hash": "72058f9253e7ed8c7243e2ff76a16d97b03d65ed", "repo_name": "sosreport/sos-collector", "repo_language": "Python"}, {"cve_id": "CVE-2018-18074", "cwe_id": "CWE-522", "repo_url": "https://github.com/requests/requests", "hash": "c45d7c49ea75133e52ab22a8e9e13173938e36ff", "repo_name": "psf/requests", "repo_language": "Python"}, {"cve_id": "CVE-2018-19787", "cwe_id": "CWE-79", "repo_url": "https://github.com/lxml/lxml", "hash": "6be1d081b49c97cfd7b3fbd934a193b668629109", "repo_name": "lxml/lxml", "repo_language": "Python"}, {"cve_id": "CVE-2018-20859", "cwe_id": "CWE-79", "repo_url": "https://github.com/edx/edx-platform", "hash": "5b144559fbdba7ff673cc1c165aa2d343e07b6bd", "repo_name": "openedx/edx-platform", "repo_language": "Python"}, {"cve_id": "CVE-2018-20954", "cwe_id": "CWE-287", "repo_url": "https://github.com/mailpile/Mailpile", "hash": "49b64f62ade9ade3dff9337c7bbc1171eab3d59e", "repo_name": "mailpile/Mailpile", "repo_language": "Python"}, {"cve_id": "CVE-2018-25045", "cwe_id": "CWE-79", "repo_url": "https://github.com/encode/django-rest-framework", "hash": "4bb9a3c48427867ef1e46f7dee945a4c25a4f9b8", "repo_name": "encode/django-rest-framework", "repo_language": "Python"}, {"cve_id": "CVE-2018-25051", "cwe_id": "CWE-79", "repo_url": "https://github.com/JmPotato/Pomash", "hash": "be1914ef0a6808e00f51618b2de92496a3604415", "repo_name": "JmPotato/Pomash", "repo_language": "Python"}, {"cve_id": "CVE-2018-25056", "cwe_id": "CWE-79", "repo_url": "https://github.com/yola/yolapi", "hash": "a0fe129055a99f429133a5c40cb13b44611ff796", "repo_name": "yola/yolapi", "repo_language": "Python"}, {"cve_id": "CVE-2018-25082", "cwe_id": "CWE-611", "repo_url": "https://github.com/zwczou/weixin-python", "hash": "e54abadc777715b6dcb545c13214d1dea63df6c9", "repo_name": "zwczou/weixin-python", "repo_language": "Python"}, {"cve_id": "CVE-2018-25087", "cwe_id": "CWE-404", "repo_url": "https://github.com/Arborator/arborator-server", "hash": "cdbdbcbd491db65e9d697ab4365605fdfab1a604", "repo_name": "Arborator/arborator-server", "repo_language": "Python"}, {"cve_id": "CVE-2018-25088", "cwe_id": "CWE-89", "repo_url": "https://github.com/blue-yonder/postgraas_server", "hash": "7cd8d016edc74a78af0d81c948bfafbcc93c937c", "repo_name": "blue-yonder/postgraas_server", "repo_language": "Python"}, {"cve_id": "CVE-2018-25091", "cwe_id": "CWE-601", "repo_url": "https://github.com/urllib3/urllib3", "hash": "adb358f8e06865406d1f05e581a16cbea2136fbc", "repo_name": "urllib3/urllib3", "repo_language": "Python"}, {"cve_id": "CVE-2018-6354", "cwe_id": "CWE-79", "repo_url": "https://github.com/formspree/formspree", "hash": "5f18eeaaa459bee9a58f70cdf7c46adb1ef34ea7", "repo_name": "formspree/formspree", "repo_language": "Python"}, {"cve_id": "CVE-2018-6596", "cwe_id": "CWE-200", "repo_url": "https://github.com/anymail/django-anymail", "hash": "c07998304b4a31df4c61deddcb03d3607a04691b", "repo_name": "anymail/django-anymail", "repo_language": "Python"}, {"cve_id": "CVE-2018-6596", "cwe_id": "CWE-200", "repo_url": "https://github.com/anymail/django-anymail", "hash": "db586ede1fbb41dce21310ea28ae15a1cf1286c5", "repo_name": "anymail/django-anymail", "repo_language": "Python"}, {"cve_id": "CVE-2018-7206", "cwe_id": "NVD-CWE-noinfo", "repo_url": "https://github.com/jupyterhub/oauthenticator", "hash": "1845c0e4b1bff3462c91c3108c85205acd3c75a2", "repo_name": "jupyterhub/oauthenticator", "repo_language": "Python"}, {"cve_id": "CVE-2018-7749", "cwe_id": "CWE-287", "repo_url": "https://github.com/ronf/asyncssh", "hash": "c161e26cdc0d41b745b63d9f17b437f073bf7ba4", "repo_name": "ronf/asyncssh", "repo_language": "Python"}, {"cve_id": "CVE-2018-7750", "cwe_id": "CWE-287", "repo_url": "https://github.com/paramiko/paramiko", "hash": "fa29bd8446c8eab237f5187d28787727b4610516", "repo_name": "paramiko/paramiko", "repo_language": "Python"}, {"cve_id": "CVE-2018-7753", "cwe_id": "CWE-20", "repo_url": "https://github.com/mozilla/bleach", "hash": "c5df5789ec3471a31311f42c2d19fc2cf21b35ef", "repo_name": "mozilla/bleach", "repo_language": "Python"}, {"cve_id": "CVE-2018-8097", "cwe_id": "CWE-94", "repo_url": "https://github.com/pyeve/eve", "hash": "f8f7019ffdf9b4e05faf95e1f04e204aa4c91f98", "repo_name": "pyeve/eve", "repo_language": "Python"}, {"cve_id": "CVE-2019-10160", "cwe_id": "CWE-172", "repo_url": "https://github.com/python/cpython", "hash": "8d0ef0b5edeae52960c7ed05ae8a12388324f87e", "repo_name": "python/cpython", "repo_language": "Python"}, {"cve_id": "CVE-2019-10682", "cwe_id": "CWE-312", "repo_url": "https://github.com/relekang/django-nopassword", "hash": "d8b4615f5fbfe3997d96cf4cb3e342406396193c", "repo_name": "relekang/django-nopassword", "repo_language": "Python"}, {"cve_id": "CVE-2019-10800", "cwe_id": "CWE-88", "repo_url": "https://github.com/codecov/codecov-python", "hash": "2a80aa434f74feb31242b6f213b75ce63ae97902", "repo_name": "codecov/codecov-python", "repo_language": "Python"}, {"cve_id": "CVE-2019-11340", "cwe_id": "CWE-20", "repo_url": "https://github.com/matrix-org/sydent", "hash": "4e1cfff53429c49c87d5c457a18ed435520044fc", "repo_name": "matrix-org/sydent", "repo_language": "Python"}, {"cve_id": "CVE-2019-12105", "cwe_id": "CWE-306", "repo_url": "https://github.com/Supervisor/supervisor", "hash": "4e334d9cf2a1daff685893e35e72398437df3dcb", "repo_name": "Supervisor/supervisor", "repo_language": "Python"}, {"cve_id": "CVE-2019-12387", "cwe_id": "CWE-74", "repo_url": "https://github.com/twisted/twisted", "hash": "6c61fc4503ae39ab8ecee52d10f10ee2c371d7e2", "repo_name": "twisted/twisted", "repo_language": "Python"}, {"cve_id": "CVE-2019-14806", "cwe_id": "CWE-331", "repo_url": "https://github.com/pallets/werkzeug", "hash": "00bc43b1672e662e5e3b8cecd79e67fc968fa246", "repo_name": "pallets/werkzeug", "repo_language": "Python"}, {"cve_id": "CVE-2020-15235", "cwe_id": "CWE-200", "repo_url": "https://github.com/ractf/core", "hash": "f3dc89b9f6ab1544a289b3efc06699b13d63e0bd", "repo_name": "ractf/core", "repo_language": "Python"}, {"cve_id": "CVE-2020-5240", "cwe_id": "CWE-863", "repo_url": "https://github.com/labd/wagtail-2fa", "hash": "ac23550d33b7436e90e3beea904647907eba5b74", "repo_name": "labd/wagtail-2fa", "repo_language": "Python"}, {"cve_id": "CVE-2021-21329", "cwe_id": "CWE-287", "repo_url": "https://github.com/ractf/core", "hash": "c57a4d186bfc586ad3edfe4dcba9f11efbf22f09", "repo_name": "ractf/core", "repo_language": "Python"}, {"cve_id": "CVE-2021-21329", "cwe_id": "CWE-287", "repo_url": "https://github.com/ractf/core", "hash": "cebb67bd16a8296121201805332365ffccb29638", "repo_name": "ractf/core", "repo_language": "Python"}]